cluster/addons/calico-policy-controller/typha-vertical-autoscaler-deployment.yaml

```diff
@@ -1,18 +1,21 @@
 kind: Deployment
-apiVersion: extensions/v1beta1
+apiVersion: apps/v1beta2
 metadata:
   name: calico-typha-vertical-autoscaler
   namespace: kube-system
   labels:
-    k8s-app: calico-typha-autoscaler
+    k8s-app: calico-typha-vertical-autoscaler
     kubernetes.io/cluster-service: "true"
     addonmanager.kubernetes.io/mode: Reconcile
 spec:
+  selector:
+    matchLabels:
+      k8s-app: calico-typha-vertical-autoscaler
```

Comment:
Will this work on upgrade from an earlier version? I'm not sure, so probably need to test it.

Worried that this will either get rejected with a "can't update selector" error or that it will leave two instances running because the selector has changed.